eng/Configurations.props

```diff
@@ -7,6 +7,8 @@
   <PropertyGroup>
     <!-- Default to portable build if not explicitly set -->
     <PortableBuild Condition="'$(PortableBuild)' == ''">true</PortableBuild>
+    <ScriptExt Condition="'$(OS)' == 'Windows_NT'">.cmd</ScriptExt>
+    <ScriptExt Condition="'$(OS)' != 'Windows_NT'">.sh</ScriptExt>
```

Comment:
This property only provides value where you want to differentiate between CMD and SH scripts. It wouldn't work for ps1 or other cases. I would prefer an unambiguous name.

Reply:
Do you have a suggestion for the name?

Reply:
ping @ViktorHofer

Reply:
I'm unsure if the overall change makes sense given that .cmd and .sh aren't the only script extensions that we use in this repo. This property would make perfect sense if there wouldn't be python, powershell and probably others in play.

Reply:
Ok, I will close this PR and issue #46235 in the next few days if we don't get other opinions.

Reply:
Nevertheless, thanks @MCCshreyas and @sandreenko for the discussion and the work.